doc/userguide/firewall/firewall-design.rst

```diff
@@ -140,6 +140,11 @@ Response (``to_client``) side:
 * ``server_handshake_done``
 * ``server_finished``
 
+ssh
+^^^
+
+Available states are listed in `ssh-hooks`.
```

Comment:
this should be a link? cc @jufajardini

Reply:
In `doc/userguide` you can do `make html`. You will need Sphinx installed though. For an example of the Ubuntu dependencies, see https://github.com/OISF/suricata/blob/master/.github/workflows/docs.yml, or for Fedora/RHEL: https://github.com/OISF/suricata/blob/master/.github/workflows/builds.yml#L68